ITISFoundation/osparc-simcore
# ✨Ui will now receive logs from dynamic-services

## What do these changes do?



Logs from all the containers started by the dynamic-sidecar are now:
- send to the UI via rabbitmq and splayed in the `Logger `
- dynamic-sidecar also sends logs to the UI for some important operations

## Related issue/s



- closes https://github.com/ITISFoundation/osparc-simcore/issues/2455
- requires https://github.com/ITISFoundation/osparc-simcore/pull/2622

## How to test



- start a local registry and push `dy-static-file-server-dybamic-sidecar-compose-spec`
- start the stack
- give yourself permissions to access the service
- create a new study and add the previous service. You will see the logs appear in the `Logger` tab 

## Checklist
